index.js

```diff
@@ -10,11 +10,9 @@ const obj = () => Object.create(null);
 
 // Prevent caching of this module so module.parent is always accurate
 delete require.cache[__filename];
-const parentDir = path.dirname(module.parent.filename);
```

Comment:
Done - btw `require-uncached` didn't do the trick, because the modules-parent was then pointing to the uncached module. I made use of your neat `clear-require`. 🍻